cmd/deploy/deploy.go

```diff
@@ -295,14 +319,23 @@ func (dc *deployCommand) runDeploy(ctx context.Context, cwd string, opts *Option
 		fmt.Sprintf("%s=%s", model.OktetoNamespaceEnvVar, okteto.Context().Namespace),
 	)
 
-	for _, command := range opts.Manifest.Deploy.Commands {
-		oktetoLog.SetStage(command)
-		if err := dc.executor.Execute(command, opts.Variables); err != nil {
-			oktetoLog.Infof("error executing command '%s': %s", command, err.Error())
-			return fmt.Errorf("error executing command '%s': %s", command, err.Error())
-		}
+	err = dc.deploy(opts)
+	if err != nil {
+		oktetoLog.LogIntoBuffer("Deployment failed: %s", err.Error())
+		cfg = app.SetStatus(cfg, app.ErrorStatus, data.Output)
+	} else {
+		oktetoLog.LogIntoBuffer("App '%s' successfully deployed", opts.Name)
```

Comment:
Check all logs to avoid mentioning `App`. This could be `"'%s' successfully deployed", opts.Name`